### DISCUSSION TITLE
SAMD21 with ST7796S support

### DISCUSSION THREAD

**Discussion by JonRobert** [Bodmer/TFT_eSPI#2611] [New processor/display support or capability request] [answered]
Is the SAMD21 XIAO support in the list of enhancements?  Can it be?
Or is one of the current options appropriate for the SEEED XIAO?

Thanks
John

**Comment by Bodmer** [comment, marked as answer]
The library may work by treating the SAMD21 as a generic processor (no processor specific optimisation). However the library is aimed at processors with greater RAM capacity so the 32K of that board will not be sufficient for many of the libraries more advanced fetures. In view of this there are no plans to add optimisation.

**Reply by JonRobert** [reply]
Thank you for your response.  I will look at the esp32 series for a better choice.
John
